File: LP1to201/VDATES.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	long long int t, d, l, r;
	cin >> t;
	while(t--) {
		cin >> d >> l>> r;
		if (d >= l && d <= r)
			cout << "Take second dose now" << endl;
		else if (d < l)
			cout << "Too Early" << endl;
		else if (d > r)
			cout << "Too Late" << endl;
	}
}

File: LP1to201/java/VDATES.java
import java.io.*;
import java.util.*;

public class VDATES {
    public static void main (String args[]) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while(t-- > 0) {
            long d, l, r;
            d = sc.nextLong();
            l = sc.nextLong();
            r = sc.nextLong();
            if (d < l)
                System.out.println("Too Early");
            else if (d > r)
                System.out.println("Too Late");
            else
                System.out.println("Take second dose now");
        }
    }
}